scripts/manage/landing_page_base.md

```diff
@@ -179,6 +179,20 @@ Update the commands and documentation in your management environment to the late
 ~/spinnaker-for-gcp/scripts/manage/connect_to_redis.sh
 ```
 
+### Restore a backup to Cloud Shell
+
+Restore a backup of the halyard configuration and deployment configuration from Cloud Source Repositories to your Cloud Shell. 
+
+```bash
+~/spinnaker-for-gcp/scripts/manage/restore_backup_to_cloud_shell.sh -p $PROJECT_ID -r $CONFIG_CSR_REPO -h GIT_HASH
+```
+
+All backups can be viewed by running:
```

Comment:
Can we just include the CSR link directly in the markdown? The landing_page_expanded.md file is generated anyway and you're using the template variables just above already. Seems a shorter path than running the script and then clicking on the link.

Reply:
Good idea. This script served more of a purpose when we were retrieving and displaying a listing from storage buckets but I agree, just putting it in the landing page doc is a better UX.